hmcts/pre-portal
# S28 2758/use stg b2c

### JIRA ticket(s)

S28-2758

### Change description

- Update local and dev envs to use staging instance of B2C
- Update github action to manage the staging B2C callback urls automatically

**QA instructions**

- Attempt to visit this PR in dev (https://pre-portal-pr-320.dev.platform.hmcts.net/ - need to be on the VPN)
- Verify that the app redirects to https://hmctsstgextid.b2clogin.com/... for login and correctly returns to this PR when successfully logging in. The Staging B2C has hardly any users - you will likely need to re-register.
